subsys/net/ip/tcp.c

```diff
@@ -1810,18 +1810,31 @@ static bool tcp_validate_seq(struct tcp *conn, struct tcphdr *hdr)
 		(net_tcp_seq_cmp(th_seq(hdr), conn->ack + conn->recv_win) < 0);
```

Comment:
Typo in commit message of `net: tcp: Add build in auto check for reordering`
s/conistend/consistent

Reply:
Fixed

Reply:
Hmm, now the commit msg says `conistent` should be `consistent`

Reply:
Sorry, fixed now